Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.5
+USER root
+WORKDIR /root
+ADD example.vimrc .vimrc
+RUN apk add --no-cache curl vim git terraform && \
+    curl -fLo ~/.vim/autoload/plug.vim --create-dirs https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim && \
+    vim +PlugInstall +qall
```

Comment:
Vim in alpine does not come with +ruby or +ruby/dyn , so the plugin won't work due to that